src/nyc_trees/nyc_trees/settings/production.py

```diff
@@ -82,3 +79,22 @@ def get_env_setting(setting):
     logger.exception('Failed to resolve TILER_HOST, %s' %
                      environ.get('TILER_HOST'))
 # END TILER CONFIGURATION
+
+# Django Storages CONFIGURATION
+try:
+    mac_metadata = get_instance_metadata()['network']['interfaces']['macs']
+    vpc_id = mac_metadata.values()[0]['vpc-id']
+
+    # The VPC id should stay the same for all app servers in a particular
+    # environment and remain the same after a new deploy, but differ between
+    # environments.  This makes it a suitable S3 bucket name
+    AWS_STORAGE_BUCKET_NAME = vpc_id
```

Comment:
I'd append or prepend something to this that indicates what we're using it for. Seeing a bucket with just a VPC ID name is hard to make sense of. Perhaps removing the `vpc-` from the VPC ID and doing something like `nyc-trees-storage-<ID>` would be better.